observability/logging/cluster-logging-deploying.adoc

```diff
@@ -7,57 +7,96 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
+[TIP]
 ====
-For new installations, use Vector and LokiStack. Elasticsearch and Fluentd are deprecated and are planned to be removed in a future release.
+You must install the {clo} *after* the log store Operator.
 ====
 
-ifdef::openshift-origin[]
-[id="prerequisites_cluster-logging-deploying"]
-== Prerequisites
-* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
-+
-If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
-endif::[]
+[id="with-the-elasticsearch-log-store_{context}"]
+== With the Elasticsearch log store
 
-//Installing the CLO via webconsole
-include::modules/cluster-logging-deploy-console.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cr-console.adoc[leveloffset=+1]
+include::snippets/logging-elastic-dep-snip.adoc[leveloffset=+1]
 
-// Install using CLI
-include::modules/cluster-logging-deploy-cli.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cli.adoc[leveloffset=+1]
+include::modules/logging-es-deploy-console.adoc[leveloffset=+2]
 
-[id="cluster-logging-deploying-postinstallation"]
-== Postinstallation tasks
+include::modules/logging-es-deploy-cli.adoc[leveloffset=+2]
 
-After you have installed the {clo}, you can configure your deployment by creating and modifying a `ClusterLogging` custom resource (CR).
+[id="with-the-loki-log-store_{context}"]
+== With the Loki log store
 
-[TIP]
+include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* Determine Loki xref:../logging/log_storage/installing-log-storage.html#loki-deployment-sizing_installing-log-storage[deployment size].
+
+* Configure xref:../logging/log_storage/installing-log-storage.adoc#logging-loki-storage_installing-log-storage[S3 compatible object storage].
+
+* Create a secret using the configuration provided for the object storage provider:
++
+.AWS
+[%collapsible]
+====
+include::modules/logging-loki-storage-aws.adoc[leveloffset=+2]
 ====
-If you are not using the Elasticsearch log store, you can remove the internal Elasticsearch `logStore` and Kibana `visualization` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources. See xref:../../observability/logging/log_storage/logging-config-es-store.adoc#cluster-logging-removing-unused-components-if-no-elasticsearch_logging-config-es-store[Removing unused components if you do not use the Elasticsearch log store].
++
+.Azure
+[%collapsible]
 ====
+include::modules/logging-loki-storage-azure.adoc[leveloffset=+2]
+====
++
+.GCP
+[%collapsible]
+====
+include::modules/logging-loki-storage-gcp.adoc[leveloffset=+2]
+====
++
+.Minio
+[%collapsible]
+====
+include::modules/logging-loki-storage-minio.adoc[leveloffset=+2]
+====
++
+.OpenShift Data Foundation
+[%collapsible]
```

Comment:
So all these collapsible includes are rendering the = in the heading: 
![image](https://github.com/openshift/openshift-docs/assets/77501157/f11939bf-83c7-46a0-b272-9822b89ee749)

As much as I think this is a cool idea to present all these options, I'm not sure if its the most user-friendly experience, especially too because all the options are no long visible in the left nav. It may need to be more like you've got here: https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage_installing-log-storage

Reply:
Or maybe add the link in the prerequsities instead? Like this:
* Create a secret using the configuration provided for the [object storage provider](https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage_installing-log-storage)